posthog/client.py

```diff
@@ -201,6 +212,9 @@ def capture(
                     msg["properties"]["$feature/{}".format(feature)] = variant
                 msg["properties"]["$active_feature_flags"] = list(feature_variants.keys())
 
+        if self.geoip_disable:
```

Comment:
Hmm, this doesn't quite work because other SDK functions, like `identify`, `alias`, `set`, don't call capture but call `_enqueue` directly.

Also, this overrides if the user sent in a property like `{"$geoip_disable": False}` which isn't what we want.

So, I'd say, move this into logic `enqueue`, and add the property if it doesn't already exist.